### ISSUE TITLE
hy_indent.hy can't be parsed

### ISSUE THREAD

**Issue by LunarArcanus** [hylang/vim-hy#12]
I couldn't get the vim plugin to work and it seemed like it complained about not being able to find the hy_indent file. Is it not being added to the module path correctly?

Furthermore, when changing to the directory of the file and running hyc on it, or importing it from within the hy repl, I get the following error:

```
    (let [l (cut (#v"getline" (first pos)) (second pos))
            ^------------------------------------------^
HyMacroExpansionError: let variable assignments must contain two items

```

This is line 31 of the module in question.

hy 0.11.1
Python 2.7.6

--
**Edit:** This issue seems to be addressed by @aisk 's pull request [here](https://github.com/hylang/vim-hy/pull/10) as that version of the file compiles. However, the interpreter still complains that it can't import the module 'vim'.

Running vim --version tells me that vim was compiled with python support and I can run :python commands as well, so not sure what the issue may be.

**Edit2:** The above was due to importing the file while not in vim, but when importing from within vim, it complains that it can't find hy instead. I'm suspecting this has something to do with the virtualenv. Going to try running it from within the system python installation (was kind of avoiding having to do that).

**Update:** The issue really was due to virtualenv. This means that the plugin won't work inside a python virtual environment.

**Comment by aisk** [comment]
It looks like some thing wrong with your vim. If it complains "can't import the module vim". Can you just run this command:

```vimscript
:python import vim; print vim
```

and see that happened?

**Comment by LunarArcanus** [comment]
@aisk I already resolved it as per the edit ( was a human error on my part because I tried importing vim from outside vim... :smile: )

I am leaving this issue open until your PR gets merged into main, I think, because there might be other people with the same problem who might not check the pull requests.

Paging @paultag

**Comment by vilmibm** [comment]
i believe this is fixed in #18
